### PR TITLE
run extract / reduce scripts in local docker image

### PR DESCRIPTION
**Pull request by camallen**
added instructions to run extract / reduce scripts via a custom locally built docker image. The docker image needed the extra PIP install to run the local code vs as a web server. 

I wasn't sure what CMD to leave in there maybe we could default to one that prints instructions if needed? Happy to rework this as needed.

### CODE REVIEW

**Comment by astopy** [comment]
It might be worth considering adding a docker-compose.yml to make it simpler to run this.

**Comment by camallen** [comment]
Can do - i had that originally but thought it might be another dependency to install, no harm in having options though :) thanks for the review :+1:

File: README.md [hunk 1]
```diff
@@ -13,6 +13,14 @@ cd aggregation-for-caesar
 pip install .
 ```
 
+Or use the docker image provided
+```
+`docker build . -f Dockerfile.bin_cmds -t aggregation_for_caesar`
+
+From the root directory of this repository, run cmds using the docker image
+docker run -it --rm --name extract_panoptes_csv -v "$PWD":/usr/src/aggregation aggregation_for_caesar python bin/extract_panoptes_csv.py --help
+```
```
Comment (astopy) [inline README.md]: Looks like the markdown formatting isn't quite right here.

File: README.md [hunk 2]
```diff
@@ -6,13 +6,35 @@ This is a collection of external reducers written for [caesar](https://github.co
...
+docker-compose -f docker-compose.local_scripts.yml build local_scripts
+```
+From the root directory of this repository, run the desirect python scripts using the docker image, e.g. `extract_panoptes_csv.py --help`
+```
+docker-compose -f docker-compose.local_scripts.yml run --rm local_scripts python bin/extract_panoptes_csv.py --help
+
+```
+**Or directly via docker**
+```
+docker build . -f Dockerfile.bin_cmds -t aggregation_for_caesar`
```
Comment (astopy) [inline README.md]: There's a stray backtick at the end of this line.

**Pull request merged**